operator/pkg/ingress/ingress_reconcile.go

```diff
@@ -276,6 +281,13 @@ func (r *ingressReconciler) buildDedicatedResources(ctx context.Context, ingress
 		}
 	}
 
+	eTP, err := annotations.GetAnnotationServiceExternalTrafficPolicy(ingress)
+	if err != nil {
+		scopedLog.WithError(err).Error("failed to get externalTrafficPolicy annotation from Ingress object")
+	} else {
```

Comment:
Else case isn't necessary as `eTP` is set to the default `Cluster` in case the annotation contained an invalid value? Hence it can be always set after checking for the error.